server/tests/e2e/maps.e2e-spec.ts

```diff
@@ -2458,4 +2458,178 @@ describe('Maps', () => {
+                return prisma.user.create({
+                    data: {
+                        steamID: steamID,
+                        alias: name,
+                        roles: { create: { verified: true, mapper: true } }
+                    }
+                });
+            }
+            function makeRun(map: Map, user: User, hash: string, ticks: number, rank: number): Promise<Run> {
+                return prisma.run.create({
+                    data: {
+                        map: { connect: { id: map.id } },
+                        user: { connect: { id: user.id } },
+                        trackNum: 0,
+                        zoneNum: 0,
+                        ticks: ticks,
+                        tickRate: 100,
+                        flags: 0,
+                        file: '',
+                        time: ticks * 100,
+                        hash: hash,
+                        rank: {
+                            create: {
+                                map: { connect: { id: map.id } },
+                                user: { connect: { id: user.id } },
+                                rank: rank,
+                                gameType: MapType.SURF
+                            }
+                        },
+                        overallStats: {
+                            create: {
+                                jumps: 1
+                            }
+                        }
+                    }
+                });
+            }
+
+            //Array just so I can use .map
+            const tmpArr = Array.from({ length: 12 }, (_, i) => i + 1);
+
+            //Need to create array of promises because of how prisma works
+            const testUserPromises = tmpArr.map((item) => makeUser(item.toString(), 'testUser' + item));
+            const testUsers: User[] = await Promise.all(testUserPromises);
+
+            //Need to create array of promises because of how prisma works
+            const testRunPromises = tmpArr.map((item) =>
+                makeRun(
+                    map4,
+                    testUsers[item - 1],
+                    '4fa6024f12494d3a99d8bda9b7a55f7d140f328' + item.toString(16),
+                    item,
+                    item
+                )
+            );
+            const testRuns: Run[] = await Promise.all(testRunPromises);
+
+            const authService = global.auth as AuthService;
+            const user7Token = (await authService.loginWeb(testUsers[6])).accessToken;
+
+            const res = await get({
+                url: `maps/${map4.id}/ranks/around`,
+                status: 200,
+                token: user7Token
+            });
+
+            let prevRank = 1;
+            for (const rank of res.body) {
+                expect(rank).toBeValidDto(MapRankDto);
+                expect(rank.rank).toBe(prevRank + 1);
+                prevRank = rank.rank;
+            }
+            expect(res.body.length).toBe(11);
+        });
```

Comment:
Please clean up all the users and runs you created at the end. In fact when we're doing DB setup for individual tests, we should probably ensure teardown happens even if the test fails. Apparently `try..finally` works (https://stackoverflow.com/a/62958360/19578983) so lets wrap all this in a `try` and in a `finally` block, teardown everything you added here (catch and ignore any failed deletions).